.github/workflows/deploy-dashboard.yml

```
@@ -0,0 +1,61 @@
+        default: 'dev'
+        options:
+          - 'dev'
+          - 'sb'
+          - 'prod'
+  workflow_call:
+    inputs:
+      environment:
+        type: string
+        description: 'Environment'
+        required: true
+        default: 'dev'
```

actionlint (1.7.4)

input "environment" of workflow_call event has the default value "dev", but it is also required. if an input is marked as required, its default value will never be used

(events)